content/en/storage-providers/advanced-configurations/sealing.md

```diff
@@ -0,0 +1,244 @@
+
+```shell
+Batch 0:
+	Message: bafy2bzacedpwysxdsg2ft3hfbwn6ayyaanivfwkx4inav3zm34hwmmwgsljkk
+	Sectors:
+		18	OK
+Batch 1:
+	Message: bafy2bzacedrx7l34ckaue7hm2ubousl3djuigyu2xw4xzywgkhttxecsm5ba2
+	Sectors:
+		19	OK
+```
+
+### CommittedCapacitySectorLifetime
+
+The available units are:
+
+```shell
+"ms": int64(Millisecond),
+ "s":  int64(Second),
+ "m":  int64(Minute),
+ "h":  int64(Hour),
+```
+
+For example, if you want to set the sector lifecycle to 180 days, you can multiply 180 days by 24 hours per day to get 4320 hours and set this value to `"4320h0m0s"`.
+
+## Storage section
+
+The storage sector controls whether the miner can perform certain sealing actions. Depending on the setup and the use of additional [seal workers]({{< relref "seal-workers" >}}), you may want to modify some of the options.
+
+```toml
+[Storage]
+  # Upper bound on how many sectors can be fetched in parallel by the storage system at a time
+  ParallelFetchLimit = 10
+  # Sealing steps that the miner can perform itself. Sometimes we have a dedicated seal worker to do them and do not want the miner to commit any resources for this.
+  AllowAddPiece = true
+  AllowPreCommit1 = true
+  AllowPreCommit2 = true
+  AllowCommit = true
+  AllowUnseal = true
+```
+
+## Fees section
+
+The fees section allows to set limits to the gas consumption for the different messages that are submitted to the chain by the miner:
+
+```toml
+[Fees]
+  # Maximum fees to pay
+  MaxPreCommitGasFee = "0.025 FIL"
+  MaxCommitGasFee = "0.05 FIL"
+  MaxTerminateGasFee = "0.5 FIL"
+  # This is a high-value operation, so the default fee is higher.
+  MaxWindowPoStGasFee = "5 FIL"
+  MaxPublishDealsFee = "0.05 FIL"
+  MaxMarketBalanceAddFee = "0.007 FIL"
+  [Fees.MaxPreCommitBatchGasFee]
+      Base = "0 FIL"
+      PerSector = "0.02 FIL"
+  [Fees.MaxCommitBatchGasFee]
+      Base = "0 FIL"
+      PerSector = "0.03 FIL"
+
+```
+
+Depending on the network congestion, the base fee for a message may grow or decrease. Your gas limits will have to be larger than the base fee for the messages to be included. A very large max fee can, however, result in the quick burning of funds when the base fee is very high, as the miner automatically submits messages during normal operation, so be careful about this. It is also necessary to have more funds available than any max fee set, even if the actual fee will be far less than the max fee set.
+
+Set the maximum cost you are willing to pay for onboarding **per** sector in `MaxPreCommitBatchGasFee.PerSector`/`MaxCommitBatchGasFee.PerSector` to avoid unexpected high costs.
+
+> Note: The current `MaxCommitBatchGasFee.PerSector` is enough to aggregate proofs for 6 sectors. Adjust this respectively according to your operation. **If the value is too low, the message may wait in the mempool for a long while. If you don't have enough funds, the message will not be sent.**
```

Comment:
If you need to add a call out use theme alerts:

```plaintext
{{< alert icon="👉" >}}
_Markdown_ and <em>HTML</em> will be rendered.
{{< /alert >}}
```

Do not use quote blocks for call outs. Quote blocks are for quotes.